tests-blackbox/common/seed-functions.ts

```diff
@@ -394,3 +398,38 @@ function generateTimestamp(options: OptionsSeedGenerateTimestamp) {
 
 	return values;
 }
+
+function generateJSONSchemaValues(schema: TestsFieldSchema, index = 0) {
+	const childKeys = Object.keys(schema);
+	const isArray = childKeys.every((key) => {
+		if (typeof key === 'number') return isInteger(key);
+		if (typeof key === 'string') {
+			return isInteger(parseInt(key));
```

Comment:
Switching to native method:
```suggestion
		if (typeof key === 'number') return Number.isInteger(key);
		if (typeof key === 'string') {
			return Number.isInteger(parseInt(key));
```